workspace/workflows/jobs.py

```diff
@@ -203,13 +203,15 @@ def get_success_rate(conclusions) -> float:
     return conclusions.count("success") / len(conclusions)
 
 
-def _summarise(header_text: str, locations: list[str]) -> str:
-    def list_conclusions(location):
-        reporter = RepoWorkflowReporter(location)
-        conclusions = reporter.get_latest_conclusions()
-        return list(conclusions.values())
+def _summarise(header_text: str, locations: list[str], skip_successful: bool) -> str:
+    unsorted = {}
+    for location in locations:
+        wf_conclusions = RepoWorkflowReporter(location).get_latest_conclusions()
+        # Use string comparison rather than success rate to avoid rounding errors
+        if skip_successful and all(c == "success" for c in wf_conclusions.values()):
```

Comment:
Is there a reason not to re-use get_success_rate?
```
if skip_successful and get_success_rate(wf_conclusions.values()) == 1:
```

Reply:
`get_success_rate` returns a float, so I was a bit worried about floating point values. Would this be a concern? I thought the current implementation is more robust.

Reply:
![image](https://github.com/user-attachments/assets/1d071f8f-a66b-4de7-9bdc-ab0b1192d916)
Okay, I am now confident that python is clever enough to know 1.0 == 1, so I will changed to the suggested code!

Reply:
Update: Using `all(c == "success" for c in wf_conclusions.values())` once more in pull #619 for a different reason: now that some failures are being hidden, using `get_success_rate` might result in division by zero.